CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 
 ### Fixes
 
+- `[jest-cli]` Support dashed args ([#7497](https://github.com/facebook/jest/pull/7497))
```

Comment:
Can you move this below the breaking changes, preferably at the bottom of this section?